.pre-commit-config.yaml

```diff
@@ -22,3 +22,4 @@ repos:
           - "pyhamcrest"
           - "kombu==5.0.2"
           - "sqlalchemy==1.3.22"
+          - "https://github.com/wazo-platform/wazo-auth-client/archive/master.zip"
```

Comment:
should we also add wazo-auth-client to the setup.py install_requires?

Reply:
Only if we want to always depend on it. It's here so that it is downloaded only for type checking and is not required nor installed at runtime. It depends what the desired behavior is.